clang/tools/dpct/DpctOptRules/cmake_script_migration_rule.yaml

```diff
@@ -2221,3 +2221,11 @@
       Out: ""
       MatchMode: Full
       RuleId: "remove_empty_library_arg"
+
+- Rule: rule_set_source_files_properties
+  Kind: CMakeRule
+  Priority: Fallback
+  CmakeSyntax: set_source_files_properties_lang
+  In: set_source_files_properties(${src_files} PROPERTIES${some_other_prop}LANGUAGE CUDA${other_props})
```

Comment:
If spaces exist between `set_source_files_properties` and `(`, you rule cannot cover it, so pls add  use:`set_source_files_properties${empty}(` to cover space scenaros.

Reply:
Done